pwiz_tools/Skyline/Model/AuditLog/AuditLogEntry.cs

```diff
@@ -1049,8 +1118,13 @@ private enum ATTR
         private enum EL
         {
             message,
+            undo_redo,
+            summary,
+            all_info,
             reason,
-            extra_info
+            extra_info,
+            en_extra_info,
+            en_hash,
```

Comment:
Again, let's drop en_ since there is only one hash and no plan to have other language-specific hashes (ja_hash, etc.)

Reply:
Okay. For "en_extra_info", would "expanded_extra_info" be fine? We already have an element called "extra_info".